spatialos-sdk/src/lib.rs

```diff
@@ -1,4 +1,37 @@
 extern crate spatialos_sdk_sys;
 
+// TODO: Where should this live? We only need it for tests in order to reduce
```

Comment:
Maybe in the internal module?

Reply:
Actually scratch that, missed the part about external tests :confused:

Reply:
Yeah, needing it both internally and externally causes problems 😬 Since it's going to be a while before any of this gets merged into master anyway, I was planning on punting on this for now (unless you have a good suggestion on how to handle it).